Website/src/rendering/src/components/FullPageSearch/OrderCloudFullPageSearch.tsx

```diff
@@ -0,0 +1,139 @@
+import { useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import FullPageSearch, { FullPageSearchResultsProps } from './FullPageSearch';
+import useOcProductList from '../../hooks/useOcProductList';
+import {
+  SearchResultsFacetClickedChangedActionPayload,
+  SearchResultsPageNumberChangedActionPayload,
+  SearchResultsSortChangedActionPayload,
+  SearchResultsKeyphraseChangedActionPayload,
+} from '@sitecore-discover/widgets';
+import { OcProductListOptions, listProducts } from '../../redux/ocProductList';
+import { clone, omit } from 'lodash';
+import { useAppDispatch } from '../../redux/store';
+import { getCategoryIdByUrlPath } from '../../helpers/CategoriesDataHelper';
+
+const OrderCloudFullPageSearch = (): JSX.Element => {
```

Comment:
Is it normal that the OrderCloud version has no price facet?

Reply:
Price faceting works a bit differently in OrderCloud. Facets must live on xp, so there can really only be one price per product to facet, this part is OK if we just assume to use the b2c price. The other more difficult challenge to overcome is that the facet options for price count each unique price as a different facet. So instead of a nice neat range 0-10, 10-20 etc that can be applied you would see all unique prices and it would be capped at 100 facets. The workaround for this would be to have a static price range defined and then apply price filtering based on that range. We could do it, but I didn't think it was worth including in this iteration, perhaps could be a fast follow. I should note that sorting by price does work though